f5lbaasdriver/test/tempest/services/clients/bigip_client.py

```
@@ -196,3 +196,34 @@ def virtual_server_has_policy(self, vs_name, policy_name, partition):
                     return True
 
         return False
+
+    def virtual_server_has_profile(self, vs_name, profile_name, partition):
+        if self.virtual_server_exists(name=vs_name, partition=partition):
```

ok. The VS is expected to exist, so I'll update the test to let the SDK raise.